app/views/layouts/application.html.haml

```diff
@@ -11,13 +11,13 @@
     - if storage_host?
       %link{ rel: 'dns-prefetch', href: storage_host }/
 
-    %link{ rel: 'icon', href: site_icon_path('favicon', 'ico') || '/favicon.ico', type: 'image/x-icon' }/
+    %link{ rel: 'icon', href: favicon_path || '/favicon.ico', type: 'image/x-icon' }/
```

Comment:
This reverts to using the wrong file format. This should be `favicon_path('ico')`.

Reply:
Thanks, good catch. Updated.